### DISCUSSION TITLE
Mac won't open because it can't be verified

### DISCUSSION THREAD

**Discussion by currentcreative** [nukeop/nuclear#987] [Releases]
Never had this happen before. I downloaded the dmg and installed it into my Applications folder just fine. Then, I double-clicked to open it and my Mac told me it could not verify that it was free from malware, so it refused to open it. Mac OS Catalina 10.15.7.

I put this under Releases just because I downloaded the latest version. I didn't download any prior versions, so maybe this always happens and there's some way around it?

**Comment by nukeop** [comment]
Mac OS is being retarded: https://disable-gatekeeper.github.io/

**Comment by UnixCro** [comment]
Since this program was not uploaded by Apple's notary service, this program does not contain a signature. As soon as this signature is missing, XProtect will issue a warning, and that is exactly the warning you described. XProtect recognizes this because an extended file attribute has been added to this file, namely the "com.apple.quarantine" flag. The operating system automatically adds this attribute to all executable files downloaded from the Internet. Only in this way can XProtect verify the signature. The situation is different with the gatekeeper, namely it also looks at the "com.apple.quarantine" flag, but only if it is on Apple's blacklist, then the program cannot start unless you change system-wide functions. Since nuclear has simply not been checked by the notary service and is not on the blacklist, you can right-click on this file and go to "Open" without worrying. That namely bypasses the message.

In addition, open-source programs cannot have any harmful content at all because, as I said, they are open-source. Everyone can see how this app was programmed.

But you can also remove the flag directly in another way, namely via the terminal with the command:

`sudo xattr -c yourfile`

**Comment by northben** [comment]
In MacOS 15.3.2 Sequoia, the error is:

> “nuclear.app” is damaged and can’t be opened. You should move it to the Trash.

Thanks to @UnixCro, the solution is easy
```
sudo xattr -c /Applications/nuclear.app
```
